### DISCUSSION TITLE
can i use chatwoot as AI?

### DISCUSSION THREAD

**Discussion by shubhamjain108-lab** [chatwoot/chatwoot#2267] [Q&A] [answered]
HI all,

I mean that how can use chatwoot in my web application with AI (artifitial intelligent) chat?
we should show question based chat with user to know our project communication.

thank you

shubham jain

**Comment by sojan-official** [comment, marked as answer]
We don't have any out of the box AI built into chatwoot at the moment. 
But you can leverage our APIs and integrate open-source NLP tools like rasa into your project

ref: https://www.chatwoot.com/blog/its-a-bot-story
https://github.com/chatwoot/chatwoot/wiki/Building-on-Top-of-Chatwoot:-Agent-Bots

**Reply by shubhamjain108-lab** [reply]
thank you for reply

**Reply by ChiefTechDev** [reply]
chatwoot used rasa?
